tests/test_sphinx_autodoc_typehints.py

```diff
@@ -554,7 +554,7 @@ def test_sphinx_output_future_annotations(app: SphinxTestApp, status: StringIO)
        Method docstring.
 
        Parameters:
-          * **x** (bool | None) -- foo
+          * **x** (bool | None) -- *foo*
```

Comment:
The [title-reference](https://docutils.sourceforge.io/docs/ref/rst/roles.html#title-reference) role that’s used by default results in a `<cite/>` node when rendered to HTML, and apparently to `*` when rendered to text.